Two integrations had test failures. upnp had the method blow up, so it would pass the test. Now that it no longer blows up we need better patching. This has been addressed.

For ZHA, I was unable to see why the device action test is failing. I am not familiar enough with the code. The error is that `radio_type` is `None` during initialization:

```
ERROR:homeassistant.config_entries:Error setting up entry Mock Title for zha
Traceback (most recent call last):
  File "/Users/paulus/dev/hass/home-assistant/homeassistant/config_entries.py", line 190, in async_setup
    hass, self
  File "/Users/paulus/dev/hass/home-assistant/homeassistant/components/zha/__init__.py", line 107, in async_setup_entry
    await zha_gateway.async_initialize()
  File "/Users/paulus/dev/hass/home-assistant/homeassistant/components/zha/core/gateway.py", line 111, in async_initialize
    radio_details = RADIO_TYPES[radio_type][ZHA_GW_RADIO]()
KeyError: None
```

CC @dmulcahey

--- a/homeassistant/components/upnp/device.py
+++ b/homeassistant/components/upnp/device.py
@@ -59,9 +59,6 @@ class Device:
         factory = UpnpFactory(requester, disable_state_variable_validation=True)
         upnp_device = await factory.async_create_device(ssdp_description)
 
-        # wrap with async_upnp_client.IgdDevice
-        from async_upnp_client.profiles.igd import IgdDevice
-
         igd_device = IgdDevice(upnp_device, None)
 
         return cls(igd_device)

--- a/tests/components/zha/test_device_action.py
+++ b/tests/components/zha/test_device_action.py
@@ -41,7 +41,7 @@ async def test_get_actions(hass, config_entry, zha_gateway):
         zha_gateway,
     )
 
-    await hass.config_entries.async_forward_entry_setup(config_entry, DOMAIN)
+    await hass.config_entries.async_forward_entry_setup(config_entry, "binary_sensor")
     await hass.async_block_till_done()
     hass.config_entries._entries.append(config_entry)
 

